custom/icds_reports/models/aggregate.py

```diff
@@ -1767,3 +1814,25 @@ class Meta(object):
 
     _agg_helper_cls = BiharApiDemographicsHelper
     _agg_atomic = False
+
+
+class BiharAPIChildVaccine(models.Model, AggregateMixin):
```

Comment:
What is the intention of this table? It looks like it mostly just replicates a lot of columns in child health monthly. It might be easier to add the few missing columns there (as you do with other ones) rather than create a new table with so much replication